Capri.Web.ViewModels/Proposal/ProposalRegistration.cs

```diff
@@ -1,20 +1,47 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
 using Capri.Database.Entities;
 
 namespace Capri.Web.ViewModels.Proposal
 {
     public class ProposalRegistration
     {
+        [Required]
+        [MinLength(5)]
+        [MaxLength(100)]
         public string TopicPolish { get; set; }
+
+        [Required]
+        [MinLength(5)]
+        [MaxLength(100)]
         public string TopicEnglish { get; set; }
+
+        [Required]
+        [MinLength(5)]
+        [MaxLength(400)]
         public string Description { get; set; }
+
+        [MaxLength(400)]
         public string OutputData { get; set; }
+
+        [MaxLength(50)]
         public string Specialization { get; set; }
-        public ProposalStatus Status { get; set; }
+
+        [Required]
+        [Range(1, 4)]
```

Comment:
So we won't store number 0?

Reply:
Why would we need to store number 0? If 0 is the upper limit, then there will be no sense in creating the proposal